README.md

```diff
@@ -107,6 +107,7 @@ To run tests, run the following command:
 ### Deployment
 
 You can deploy this project using:
+https://azeezumoru.github.io/Abdul-portfolio/
 
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/224512526-52eb5bc1-14d5-406e-8fda-d538d7b2e8f3.png)

**[OPTIONAL]**

- [ ] It would be nice if you make your Getting Started section as shown below to make your readme file look attractive and descriptive and professional 😊

## 💻 Getting Started <a name="getting-started"></a>


To get a local copy up and running, follow these steps.

### Prerequisites

In order to run this project you need:

You need to have npm installed on your PC first.


### Setup

Clone this repository to your desired folder:

* Copy the URL [https://github.com/azeezumoru/Abdul-portfolio.git].
* In Visual Studio Code “Terminal” change the current working directory to the location where you want the cloned directory.
* Type git clone in the terminal, paste the URL you copied earlier, and press “enter” to create your local clone.
```
git clone https://github.com/azeezumoru/Abdul-portfolio.git
```
### Install

Install this project with:
```
npm install
```

### Usage

To run the project, execute the following command:
```
npm start or run on the live server
```

### Run tests

To run tests, run the following command:
```
npm test
```
### Deployment

You can deploy this project using:
```
npm run build
```